estimates.adoc

```diff
@@ -0,0 +1,13 @@
+# Tic tac toe (OX)
+
+|===
+|*Date*|*Pessimistic*|*Realistic*|*Optimistic*
+|18.12.2019
+|Creating project, adding to remote repository with collaborators + CRC cards + flow diagram
+|+ Drawing board with given parameters
+|+ Player input validation + Asking players who starts
+|19.12.2019
```

Comment:
Try planning for whole week and then you'll see earlier how far are you getting and what stops you. That will help with re-estimation.